src/components/ShoppingCart/dropDownButton.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+import { GrDown, GrUp } from "react-icons/gr";
+import { IconButton } from "@chakra-ui/react";
+import { Collapse } from "@chakra-ui/react";
+
+type quotationDetail = {
+  content: string;
+};
+const DropDownButton = ({ content }: quotationDetail) => {
+  const [show, setShow] = useState(false);
+
+  const handleToggle = () => setShow(!show);
+
+  const scroll = "scroll";
+  const hidden = "hidden";
+  const noWrap = "nowrap";
+  const normal = "normal";
+  const clip = "clip";
+  const ellipsis = "ellipsis";
+
+  return (
+    <div
+      style={{
+        width: "100%",
+        display: "flex",
+        flexDirection: "row",
+        justifyContent: "center",
+        alignContent: "center",
+        overflow: "auto",
+      }}
+    >
+      <Collapse
+        startingHeight={25}
+        in={show}
+        style={{
+          overflowY: show ? scroll : hidden,
+          userSelect: "none",
+          whiteSpace: show ? normal : noWrap,
+          textOverflow: show ? clip : ellipsis,
+        }}
+        data-testid="testShow"
+      >
+        {content}
+      </Collapse>
```

Comment:
<img width="319" alt="Screen Shot 2022-09-26 at 7 40 04 pm" src="https://user-images.githubusercontent.com/76163407/192244932-1d3cb172-ca77-45fa-adb2-41fdff24d027.png">


If the content of the quotation details can be fully displayed on the container, is there no need for the drop-down arrow?
<br/>
You can see my screenshot, when the quotation content can be displayed normally, I click on the drop-down arrow, the container will have a scroll bar, but it has no effect

Reply:
It is the old one.  Maybe you should pull my latest branch.